Apps/phcywarectix/ctix.json

```diff
@@ -0,0 +1,650 @@
+{
+    "appid": "9c859a3a-310a-43a2-9925-aaf84bc6abbf",
+    "name": "Cyware CTIX",
+    "description": "Cyware Threat Intel eXchange is an intelligent client-server intelligence exchange that provides subscriber with full Threat Intel collection management from multiple internal and external sources",
+    "type": "Threat Intelligence",

```

Comment:
This should be "threat intel" as its approved app type.

Reply:
Fixed